From c9c0f73ee228d202d4a7d54f0d51bcdab2346f41 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 24 Mar 2020 17:53:53 +0100
Subject: [PATCH 1293/1851] dt-bindings: versal: Add new PM_DEV_AI and
 PM_DEV_GPIO_PMC macros

commit 46b6949d91b113a8c506a2f5948bf25ad985f20f from
https://github.com/Xilinx/linux-xlnx.git

Extend header with new macros and also update years.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/dt-bindings/power/xlnx-versal-power.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/dt-bindings/power/xlnx-versal-power.h b/include/dt-bindings/power/xlnx-versal-power.h
index a869194b965a..1b75175edce5 100644
--- a/include/dt-bindings/power/xlnx-versal-power.h
+++ b/include/dt-bindings/power/xlnx-versal-power.h
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: GPL-2.0 */
 /*
- *  Copyright (C) 2019 Xilinx, Inc.
+ *  Copyright (C) 2019 - 2020 Xilinx, Inc.
  */
 
 #ifndef _DT_BINDINGS_VERSAL_POWER_H
@@ -25,6 +25,7 @@
 #define PM_DEV_SWDT_FPD				(0x18224029U)
 #define PM_DEV_OSPI				(0x1822402aU)
 #define PM_DEV_QSPI				(0x1822402bU)
+#define PM_DEV_GPIO_PMC				(0x1822402cU)
 #define PM_DEV_SDIO_0				(0x1822402eU)
 #define PM_DEV_SDIO_1				(0x1822402fU)
 #define PM_DEV_RTC				(0x18224034U)
@@ -36,5 +37,6 @@
 #define PM_DEV_ADMA_5				(0x1822403aU)
 #define PM_DEV_ADMA_6				(0x1822403bU)
 #define PM_DEV_ADMA_7				(0x1822403cU)
+#define PM_DEV_AI				(0x18224072U)
 
 #endif
-- 
2.31.1

